maas-schemas/schemas/maas-backend/subscriptions/subscriptions-retrieve/request.json

```diff
@@ -3,12 +3,15 @@
   "description": "Request schema for subscriptions-retrieve",
   "type": "object",
   "properties": {
-    "identityId": {
+    "customerId": {
```

Comment:
This is a breaking change, so bump the major version.

Reply:
IdentityId still exists :D

Reply:
Making customerId required is breaking change perhaps?